src/document-templates.ts

```diff
@@ -5,15 +5,18 @@ export async function getDocumentTemplateNamesForFlow(
   client: GraphQLClient,
   flowId: string
 ): Promise<string[]> {
-  const { flow_document_templates: response } = await client.request(gql`
-    query GetDocumentTemplateNamesForFlow($flowId: uuid!) {
-      flow_document_templates(where: {flow: {_eq: $flowId}}) {
-        document_template
-      }
-    }`,
-    { flowId }
-  );
-  return response.map(
+  const response: { flow_document_templates: [{ document_template: string }] } =
```

Comment:
The type declarations are updated here to properly unpack the returned `unkown` type returned by the updated GraphQLClient.